<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RestorationDialog</class>
 <widget class="QDialog" name="RestorationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Restoration</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="itemListWidget">
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>These items couldn't be restored on their own.
Please select the destination folder.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nextItemButton">
          <property name="text">
           <string>Next item</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="applyToAllButton">
          <property name="text">
           <string>Apply to all items</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="treeView">
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="headerDefaultSectionSize">
        <number>21</number>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>itemListWidget</tabstop>
  <tabstop>treeView</tabstop>
  <tabstop>nextItemButton</tabstop>
  <tabstop>applyToAllButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RestorationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RestorationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
